web/src/filter.ts

```diff
@@ -878,6 +885,14 @@ export class Filter {
         return possible_prefix + "all public channels";
     }
 
+    static describe_archived_channels(negated: boolean): string {
+        const possible_prefix = negated ? "exclude " : "";
+        if (page_params.is_spectator || current_user.is_guest) {
+            return possible_prefix + "all archived channels that you can view";
+        }
+        return possible_prefix + "all archived channels";
```

Comment:
I think this one might just want to be "archived channels", without the "all" or conditional. Unlike "public channels", normal users can't access random private channels.

Reply:
Yeah, let's write this as just a fixed "archived channels" string, that's the correct description for this operator.